.github/actions/build_pandas/action.yml

```diff
@@ -24,9 +30,10 @@ runs:
 
     - name: Build Pandas
       run: |
+        export CFLAGS="$CFLAGS ${{ inputs.cflags_adds }}"
         if [[ ${{ inputs.editable }} == "true" ]]; then
-          pip install -e . --no-build-isolation -v --no-deps
+          pip install -e . --no-build-isolation -v --no-deps ${{ inputs.meson_args }}
```

Comment:
If we like this design we could also make `-e` a part of the meson_args input

Reply:
That's mostly a `pip` argument rather than a meson/meson-python one. Probably better to keep `meson_args` for `config_settings` arguments only?

Reply:
Ah good point